css/main.css

```diff
@@ -29,6 +29,10 @@ input[type="number"] {
 	max-width: 6em;
 }
 
+#heures_remunerees_volume {
```

Comment:
You can avoid setting the `id` with a CSS attribute query: `[name="heures_remunerees_volume"]`.